.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -204,6 +208,7 @@ steps:
         --deploy-mg-extra-params="${{ parameters.DEPLOY_MG_EXTRA_PARAMS }}" \
         --common-extra-params="${{ parameters.COMMON_EXTRA_PARAMS }}" \
         --vm-type ${{ parameters.VM_TYPE }} --num-asic ${{ parameters.NUM_ASIC }} \
+        --ptf_image_tag ${{ parameters.PTF_IMAGE_TAG }} \
```

Comment:
@lerry-lee the ansible task for [add-topo](https://github.com/sonic-net/sonic-mgmt/blob/38f202c197046e9d419266d6958d3ec6bc5ae7fd/ansible/roles/vm_set/tasks/add_topo.yml#L160), [renumber-topo](https://github.com/sonic-net/sonic-mgmt/blob/38f202c197046e9d419266d6958d3ec6bc5ae7fd/ansible/roles/vm_set/tasks/renumber_topo.yml#L15) uses `ptf_imagetag` (not `ptf_image_tag`) note the missing underscore. Can you confirm this would be passed into `ptf_imagetag` in ansible?

Reply:
@opcoder0 yes, this `ptf_image_tag` parameter is only defined in test plan. And in the restart ptf command, it indeed use `ptf_imagetag`.